python_course_1604/class_01/tests/test_04.py

```diff
@@ -12,7 +14,9 @@
 
 
 @skip_if_exercize_not_started(exercize)
-@unittest.skip('It brokes coverage.')
+@unittest.skipIf(
```

Comment:
Ha. I knew I had forgotten something ☺

From: Federico Ressi [mailto:notifications@github.com]
Sent: Wednesday, May 4, 2016 11:20 AM
To: FedericoRessi/pythoncourse <pythoncourse@noreply.github.com>
Cc: Stedje, Gustav <gustav.stedje@intel.com>; Comment <comment@noreply.github.com>
Subject: Re: [FedericoRessi/pythoncourse] Gstedje (#20)


In python_course_1604/class_01/tests/test_04.py<https://github.com/FedericoRessi/pythoncourse/pull/20#discussion_r62017580>:

> @@ -12,7 +14,9 @@

>

>

>  @skip_if_exercize_not_started(exercize)

> -@unittest.skip('It<mailto:-@unittest.skip('It> brokes coverage.')

> +@unittest.skipIf<mailto:+@unittest.skipIf>(

This is not working:

E   NameError: name 'socket' is not defined

That means you have to import socket module at the top of the exercise:

import socket

—
You are receiving this because you commented.
Reply to this email directly or view it on GitHub<https://github.com/FedericoRessi/pythoncourse/pull/20/files/650f223ea72160b74228539582a34ed247c0fb76..1809ef13da2e1038db304b564d984b4dfc66fa24#r62017580>
--------------------------------------------------------------
Intel Research and Development Ireland Limited
Registered in Ireland
Registered Office: Collinstown Industrial Park, Leixlip, County Kildare
Registered Number: 308263


This e-mail and any attachments may contain confidential material for the sole
use of the intended recipient(s). Any review or distribution by others is
strictly prohibited. If you are not the intended recipient, please contact the
sender and delete all copies.